Dockerfile.centos

```diff
@@ -0,0 +1,56 @@
+FROM centos:7 AS build-env
```

Comment:
We should reverse the name so that we get syntax highlighting.
That or add the magic vim and emacs comments to set the filetype.